playwright/api/basics.ts

```diff
@@ -51,6 +51,8 @@ export class BasicApiService {
     });
     const loginResponseJson = await loginResponse.json();
 
-    return loginResponseJson as ApiResponse<TokenCreateResponse>;
+    expect(loginResponseJson.data.tokenCreate.errors).toEqual([]);
```

Comment:
this line breaks SALEOR_137 where after deactivating the user we check API response that user cannot login if account is not active

```
test("TC: SALEOR_137 Admin User should be able to deactivate other user @e2e @staff-members", async () => {
  await staffMembersPage.goToStaffDetailsPage(USERS.userToBeDeactivated.id);
  await staffMembersPage.clickIsActiveCheckbox();
  await staffMembersPage.clickSaveButton();
  await staffMembersPage.basePage.expectSuccessBanner();
  await expect(await staffMembersPage.isActiveCheckbox.isChecked()).toEqual(false);

  const loginViaApiDeactivatedUserResponse = await basicApiService.logInUserViaApi({
    email: USERS.userToBeDeactivated.email,
    password: process.env.E2E_PERMISSIONS_USERS_PASSWORD!,
  });

  await expect(loginViaApiDeactivatedUserResponse.data.tokenCreate.errors[0].code).toEqual(
    "INACTIVE",
  );
});
```

Reply:
fixed https://github.com/saleor/saleor-dashboard/pull/5065/commits/a1af565932df563c4697dfd404e59ce8eae9545e